packages/bcp/src/cosmosconnection.spec.ts

```diff
@@ -41,30 +41,26 @@ function makeRandomAddress(): Address {
   return Bech32.encode(defaultAddressPrefix, Random.getBytes(20)) as Address;
 }
 
-const faucet = {
-  mnemonic:
-    "economy stock theory fatal elder harbor betray wasp final emotion task crumble siren bottom lizard educate guess current outdoor pair theory focus wife stone",
-  path: HdPaths.cosmosHub(0),
-  pubkey: {
+const alice = {
+  address0: "cosmos14qemq0vw6y3gc3u3e0aty2e764u4gs5le3hada" as Address,
+  pubkey0: {
     algo: Algorithm.Secp256k1,
-    data: fromBase64("A08EGB7ro1ORuFhjOnZcSgwYlpe0DSFjVNUIkNNQxwKQ") as PubkeyBytes,
+    data: fromBase64("A9cXhWb8ZpqCzkA8dQCPV29KdeRLV3rUYxrkHudLbQtS") as PubkeyBytes,
   },
-  address: "cosmos1pkptre7fdkl6gfrzlesjjvhxhlc3r4gmmk8rs6" as Address,
 };
 
 /**
  * We use a different test account than the faucet, since BCP errors when transactions
  * with multiple messages are found.
  */
 const bob = {
-  mnemonic:
-    "economy stock theory fatal elder harbor betray wasp final emotion task crumble siren bottom lizard educate guess current outdoor pair theory focus wife stone",
-  path: HdPaths.cosmosHub(4),
+  mnemonic: "remain fragile remove stamp quiz bus country dress critic mammal office need",
```

Comment:
Why the change here?

Ah, this is to keep in sync with other iov/bcp code rather than in sync with other cosm(wasm)-js code, right?

Reply:
A few PRs ago, we decided to use the mnemonic "economy stock theory fatal elder harbor …" for the faucet package only. The usage here was forgotten. With browser testing and the new independence of faucet, bcp, cosmwasm, we get new parallel running conflicts that did not occur in the node testing by pure luck.